app.js

```diff
@@ -66,13 +77,24 @@ async function warmupAgendas(agendas) {
 
 async function warmupAgenda(agenda, allowedGroupHeader) {
   const urls = await getAgendaitemsRequestUrls(agenda);
-  for (let url of urls) {
-    await fetch(url, {
-      method: "GET",
-      headers: {
-        "mu-auth-allowed-groups": allowedGroupHeader,
-      },
+  try {
+    await Promise.all(
+      urls.map(async (url) => {
+        return fetch(url, {
+          method: "GET",
+          headers: {
+            "mu-auth-allowed-groups": allowedGroupHeader,
+          },
+        }).catch((error) => {
+          throw error;
+        });
+      })
+    ).catch((error) => {
+      throw error;
```

Comment:
Wat is de meerwaarde van deze `catch`-blokken?

Reply:
Ik had last met `UnhandledPromiseRejectionWarning: Unhandled promise rejection` zonder die catch blokken, die try catch is niet voldoende als promises falen (door cache errors)

Reply:
mogelijk is slechts 1 van die catchen nodig, moet ik dan nog eens bekijken